### PR TITLE
change some looks

### PR DESCRIPTION
**Pull request by eitheingiswe**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2809,7 +2809,6 @@
       "resolved": "https://registry.npmjs.org/@styled-system/background/-/background-5.1.2.tgz",
       "integrity": "sha512-jtwH2C/U6ssuGSvwTN3ri/IyjdHb8W9X/g8Y0JLcrH02G+BW3OS8kZdHphF1/YyRklnrKrBT2ngwGUK6aqqV3A==",
       "dev": true,
-      "optional": true,
```
Comment (sebastianquek) [inline package-lock.json]: Not sure why there are changes here, please discard your changes to this file

File: src/templates/customTemplate/customTemplate.tsx [hunk 2]
```diff
@@ -33,95 +33,120 @@ const RatingGroupDetails = styled.div`
...
+  font-weight: 700;
 `;
 
 const Profile = styled.div`
   ${flexColumnCenter}
-  height: 50%;
-  padding-left: 20px;
-  padding-right: 20px;
+  height: 47%;
+  background: url(${constants.bgMap}) center ;
```
Comment (sebastianquek) [inline src/templates/customTemplate/customTemplate.tsx]:
```suggestion
  background: url(${constants.bgMap}) center;
```

File: src/templates/customTemplate/customTemplate.tsx [hunk 3]
```diff
@@ -33,95 +33,120 @@ const RatingGroupDetails = styled.div`
...
+  align-items: center;
   background-color: ${constants.colorWhite};
+  flex-shrink: 0;
+  height: 125px;
+  width: 100px;
+  overflow: hidden;
 `;
 
-const LicenseDetails = styled.div`
+const LicenseDetails = styled.div`
```
Comment (sebastianquek) [inline src/templates/customTemplate/customTemplate.tsx]:
```suggestion
const LicenseDetails = styled.div`
```

File: src/templates/customTemplate/customTemplate.tsx [hunk 4]
```diff
@@ -33,95 +33,120 @@ const RatingGroupDetails = styled.div`
...
+  position: absolute;  
+  width: 36px;
+  height: 36px;
+  border-radius: 50%;
+  background-color: ${constants.colorMaroon};
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+	box-shadow: 2px 2px 4px #000;
```
Comment (sebastianquek) [inline src/templates/customTemplate/customTemplate.tsx:131]:
```suggestion
  box-shadow: 2px 2px 4px #000;
```

File: src/templates/customTemplate/customTemplate.tsx [hunk 5]
```diff
@@ -190,62 +215,70 @@ export const CustomTemplate: FunctionComponent<TemplateProps<CustomTemplateCerti
       <FlipCard isFlipped={isFlipped}>
         <FlipCardFace isFront={true}>
           <Profile>
+            <LicenseName className="text-uppercase">
+              {document.name} (UAPL)
```
Comment (sebastianquek) [inline src/templates/customTemplate/customTemplate.tsx]: Is hardcoding `(UAPL)` the best way? What if the licence name changes?

**Review by sebastianquek — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (eitheingiswe) [reply package-lock.json]: sure.

File: src/templates/customTemplate/customTemplate.tsx (hunk 5 above)
Comment (eitheingiswe) [reply src/templates/customTemplate/customTemplate.tsx]: oh no, we don't need it. i am able to add this (UAPL) in json file

**Review by sebastianquek — APPROVED** [review]

**Pull request merged**